activities/communication/blogging.md

```diff
@@ -0,0 +1,87 @@
+
+This approach reduces our [future content debt](https://18f.gsa.gov/2016/05/19/content-debt-what-it-is-where-to-find-it-and-how-to-prevent-it-in-the-first-place/) and avoids undermining our position of authority - for example, we'll never have to change a blogpost to say that due to a government policy shift, an expert we’ve interviewed is no longer in post (which might cast doubt on that expert’s still-relevant advice).
+
+## Before you start
+
+1. Since none of us work work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+
+* understand your goals and audience for the blogpost
+* establish the scope of your blogpost (to avoid duplication with other planned work)
+* schedule publication of your blogpost so it fits best with other planned communications
+
+2. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+
+3. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
+
+## Write your blogpost
+
+Write your draft blogpost using a collaborative document tool. This lets you share it for pair writing, factchecking, feedback or proofreading.
+
+Don't hesitate to involve other people in your writing process - we want you to succeed, and we know how helpful an external perspective can be once something starts to feel like a problem.
+
+Read the [guide on writing community call blogposts](community-call-blogposts.md).
+
+After writing, please share with the communications coordinator for polishing.
+
+## How to create a blogpost on the Foundation’s blog
+
+1. Create a branch for your new blogpost within in the [blog repo](https://github.com/publiccodenet/blog).
+
+2. Create a new file for your blogpost within [`_posts/` on the blog repository](https://github.com/publiccodenet/blog/tree/develop/_posts). Name your file `YEAR-MONTH-DAY-title.md`.
+
+3. Add the frontmatter.
+
+```
+---
+title:  "Title”
+date: YYYY-MM-DD
+author: [put your name]
+type: blogpost
+category: [our current categories are: news and community call]
+excerpt: [unformatted text you want to appear on the index]
+---
+```
+
+4. Add any images to [`_assets/` on the blog repository](https://github.com/publiccodenet/blog/tree/develop/assets). Link to the asset directly in text.
+
+```
+![alternative text](/assets/image.jpg)
+```
+
+[Make the text alternative for your image useful](https://www.w3.org/WAI/tips/writing/#write-meaningful-text-alternatives-for-images).
+
+All blogposts should include at least one relevant image, to make them more engaging.
+
+5. Approvals: tag the communications coordinator on the pull request for your blogpost for a final proofread.
```

Comment:
This can be also without the numbers or alternatively with headings for each step. The text already has an order.